ley.config.mjs

```diff
@@ -1,13 +1,43 @@
-const options = {};
+import fs from 'node:fs';
+import dotenv from 'dotenv';
 
-if (process.env.POSTGRES_URL) {
-  options.ssl = true;
+// End replacement for dotenv-safe
+export function config() {
+  if (process.env.FLY_IO) return;
```

Comment:
let's try to avoid multiple separate checks if possible - let's try to combine the checks into 2 blocks (dev or production)

Reply:
9fa36ec